src/log/types.ts

```diff
@@ -12,6 +12,9 @@ export enum EventType {
 
   oauth2BadRedirect = 11,
   generateAccessToken = 12,
+
+  accountLocked = 13,
+  loginFailedAccountLocked = 14,
```

Comment:
I think it's worth adding some comments to these 2 lines so it's a bit clearer what the difference is.